_posts/2017-10-15-Assembling-a-Multicopter.md

```diff
@@ -0,0 +1,318 @@
+---
```

Comment:
One small change. Add links of each item from hobby-king or if possible, a seller from India. Would be very cool

Reply:
Sure. Working on it.